keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -146,13 +146,110 @@ and are not requesting additional changes.
 * Test coverage includes non-preempting use cases.
 * Conformance requirements for non-preempting PriorityClasses are agreed upon.
 
-## Testing Plan
+### Risks and Mitigations
+
+The new feature may malfuction,
+or existing preemption functionality may be impaired.
+New tests (covering both nonpreepting workloads and mixed workloads),
+and the existing preempting PriorityClass tests should be used to prove stability.
+
+## Design Details
+### Testing Plan
 Add detailed unit and integration tests for nonpreempting workloads.
 
 Add basic e2e tests, to ensure all components are working together.
 
 Ensure existing tests (for preempting PriorityClasses) do not break.
 
+### Graduation Criteria
+#### Alpha (v1.15):
+
+- [x] Support NonPreemptingPriority in PriorityClasses
+
+#### Beta (v1.19):
+
+- [ ] Add integration test for NonPreemptingPriority.
+- [ ] Graduate NonPreemptingPriority to Beta.
+- [ ] Update documents to reflect the changes.
+
+
+
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: NonPreemptingPriority
+    - Components depending on the feature gate: 
+      - kube-apiserver 
+      - kube-scheduler
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  Yes
```

Comment:
Answer with Yes should be provided with some explanatory words. (applies elsewhere)